server/controller/triproutes.js

```diff
@@ -1,12 +1,12 @@
 const express = require('express');
 const router = express.Router();
-const middleware = require('../handlers/middleware');
+const middleware = require('../middleware');
 const tripHandler = require('../handlers/tripHandler');
 
 // All the routes for trips
-router.put('/:tid', middleware.checkToken, tripHandler.updateTrip);
-router.delete('/:tid', middleware.checkToken, tripHandler.deleteTrip);
-router.get('/', middleware.checkToken, tripHandler.getTrips);
-router.post('/', middleware.checkToken, tripHandler.addTrip);
+router.get('/get', middleware.checkToken, tripHandler.getTrips);
```

Comment:
This file is no longer used. Should it be deleted?

Reply:
Which file? The 'triproutes' file? I've removed our general 'approutes' file.

Reply:
Yes: there is both a triproutes and a tripRoutes file right now (we want the latter, I assume)

Reply:
Bizarre. It wasn't showing up on my local branch, but it was on the remote one. It's now been removed.